device_name:
- gs
- scorp
- drone

device_mac:
- 70:b3:d5:1f:71:1c
- 70:b3:d5:1f:71:30
- 70:b3:d5:1f:71:35

device_ip:
- 192.168.150.100
- 192.168.150.101
- 192.168.150.102